src/extra/Markdown/index.tsx

```diff
@@ -1,743 +1,81 @@
-		font-size: 11px;
-		line-height: 10px;
-		color: #444d56;
-		vertical-align: middle;
-		background-color: #fafbfc;
-		border: solid 1px #c6cbd1;
-		border-bottom-color: #959da5;
-		border-radius: 3px;
-		box-shadow: inset 0 -1px 0 #959da5;
-	}
-
-	h1,
-	h2,
-	h3,
-	h4,
-	h5,
-	h6 {
-		margin-top: 24px;
-		margin-bottom: 16px;
-		font-weight: 600;
-		line-height: 1.25;
-	}
-
-	h1 .octicon-link,
-	h2 .octicon-link,
-	h3 .octicon-link,
-	h4 .octicon-link,
-	h5 .octicon-link,
-	h6 .octicon-link {
-		color: #1b1f23;
-		vertical-align: middle;
-		visibility: hidden;
-	}
-
-	h1:hover .anchor,
-	h2:hover .anchor,
-	h3:hover .anchor,
-	h4:hover .anchor,
-	h5:hover .anchor,
-	h6:hover .anchor {
-		text-decoration: none;
-	}
-
-	h1:hover .anchor .octicon-link,
-	h2:hover .anchor .octicon-link,
-	h3:hover .anchor .octicon-link,
-	h4:hover .anchor .octicon-link,
-	h5:hover .anchor .octicon-link,
-	h6:hover .anchor .octicon-link {
-		visibility: visible;
-	}
-
-	h1 {
-		padding-bottom: 0.3em;
-		font-size: 2em;
-		border-bottom: 1px solid #eaecef;
-	}
-
-	h2 {
-		padding-bottom: 0.3em;
-		font-size: 1.5em;
-		border-bottom: 1px solid #eaecef;
-	}
-
-	h3 {
-		font-size: 1.25em;
-	}
-
-	h4 {
-		font-size: 1em;
-	}
-
-	h5 {
-		font-size: 0.875em;
-	}
-
-	h6 {
-		font-size: 0.85em;
-		color: #6a737d;
-	}
-
-	ul,
-	ol {
-		padding-left: 2em;
-	}
-
-	ul ul,
-	ul ol,
-	ol ol,
-	ol ul {
-		margin-top: 0;
-		margin-bottom: 0;
-	}
-
-	li > p {
-		margin-top: 16px;
-	}
-
-	li + li {
-		margin-top: 0.25em;
-	}
-
-	dl {
-		padding: 0;
-	}
-
-	dl dt {
-		padding: 0;
-		margin-top: 16px;
-		font-size: 1em;
-		font-style: italic;
-		font-weight: 600;
-	}
-
-	dl dd {
-		padding: 0 16px;
-		margin-bottom: 16px;
-	}
-
-	table {
-		display: block;
-		width: 100%;
-		overflow: auto;
-	}
-
-	table th {
-		font-weight: 600;
-	}
-
-	table th,
-	table td {
-		padding: 6px 13px;
-		border: 1px solid #dfe2e5;
-	}
-
-	table tr {
-		background-color: #fff;
-		border-top: 1px solid #c6cbd1;
-	}
-
-	table tr:nth-child(2n) {
-		background-color: #f6f8fa;
-	}
+import Heading from '../../components/Heading';
+import Link from '../../components/Link';
+import unified from 'unified';
+import remarkParse from 'remark-parse';
+import remarkRehype from 'remark-rehype';
+import rehypeReact from 'rehype-react';
+import highlight from 'rehype-highlight';
+import sanitize from 'rehype-sanitize';
+import raw from 'rehype-raw';
+import Divider from '../../components/Divider';
+import styled from 'styled-components';
+import gh from 'hast-util-sanitize/lib/github';
 
-	img {
-		max-width: 100%;
-		box-sizing: content-box;
-		background-color: #fff;
-	}
+export { gh as defaultSanitizerOptions };
 
-	code {
-		padding: 0;
-		padding-top: 0.2em;
-		padding-bottom: 0.2em;
-		margin: 0;
+const MarkdownWrapper = styled(Txt)`
+	& code {
 		background-color: rgba(27, 31, 35, 0.05);
+		padding: 0.2em 0px;
+		margin: 0px;
 		border-radius: 3px;
+		font-family: 'Ubuntu Mono', 'Courier New', monospace;
```

Comment:
Why Ubuntu Mono? Do we need all these custom properties in the wrapper? It feels a bit random TBH

Reply:
These styles are for inline code blocks, which were there before and we currently lack a codeblock component in rendition for styles.

Reply:
I think you can use the exported `monospace` const from `theme.ts` here.